app/controllers/courses/vacancies_controller.rb

```diff
@@ -13,11 +13,16 @@ def update
       params.dig(:course, :site_status_attributes)
         &.values&.each do |vacancy_status|
           site_status            = find_site_status vacancy_status[:id]
-          site_status.vac_status = VacancyStatusDeterminationService.call(
+          # Set all site_status.vac_status to 'no_vacancies' if radio button is checked
+          if params[:course][:has_vacancies] == 'false'
+            site_status.vac_status = 'no_vacancies'
+          else
+            site_status.vac_status = VacancyStatusDeterminationService.call(
             vacancy_status_full_time: vacancy_status[:full_time],
             vacancy_status_part_time: vacancy_status[:part_time],
             course:                   @course
           )
```

Comment:
Whitespace ... lines 21-24 need to be indented in to match the change to line 20.